build/config/PublishData.json

```diff
@@ -27,14 +27,21 @@
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.7p2", "dev15.7" ]
-        },      
+        },
         "dev15.7-preview1-vs-deps": {
             "nugetKind": "PerBuildPreRelease",
             "version": "2.8.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.7p1"]
-        },     
+        },
+        "dev15.8-preview1-vs-deps": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.9.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev15.8p1"]
```

Comment:
you need to delete the existing dev15.8p1 tag which is on line 8

Reply:
Thanks, fixed.